lib/devtools/config.rb

```diff
@@ -150,7 +150,7 @@ class Mutant < self
     # Devtools configuration
     class Devtools < self
       FILE = 'devtools.yml'.freeze
-      DEFAULT_UNIT_TEST_TIMEOUT = 0.1  # 100ms
+      DEFAULT_UNIT_TEST_TIMEOUT = 0.1 # 100ms
```

Comment:
Yeah sure I'll take a look.  It is from Style/ExtraSpacing and the discussion around what was added is here https://github.com/bbatsov/rubocop/issues/2035. Not sure yet about config